mat64/matrix.go

```diff
@@ -274,6 +274,79 @@ func Row(dst []float64, i int, a Matrix) []float64 {
 	return dst
 }
 
+// Cond returns the condition number of the given square matrix under the given
+// norm. The condition number must be based on the 1-norm, 2-norm or ∞-norm.
+//
+// BUG(btracey): The computation of the 1-norm and ∞-norm for non-square matrices
+// is innacurate (though typically the right order of magnitude). See
```

Comment:
Use comma instead of parentheses; s/ (though typically the right order of magnitude)/. although is typically the right order of magnitude/.